src/styles/parts/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  width: 200px;
+  border: 1px solid $color-white-color;
+  border-radius: 5px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+
+  &__apple {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__descript {
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    color: $color-main-accent;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+
+    color: $color-second-text;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    margin-bottom: 24px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    &-number {
+      color: $color-main-accent;
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-text {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+
+      color: $color-second-text;
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $color-main-accent;
+    }
+  }
+
+  &__buy {
+    width: 100%;
+    padding: 12px 0;
+
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    text-decoration: none;
+    color: $color-white-color;
+    background-color: $color-accent-blue;
+
+    display: inline-block;
+    text-align: center;
+
+    text-transform: uppercase;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+
+    &:hover {
+      background-color: $color-white-color;
+      border-color: $color-accent-blue;
+      color: $color-accent-blue;
+    }
+  }
+
+  &:nth-child(-n+#{$cards-number * 2}) {
+    margin-right: 48px;
+    margin-bottom: 46px;
```

Comment:
I think that it still counts as adding margin on BEM block and should be moved to any BEM element of container like for example page__card, just can't be on BEM block.